### ISSUE TITLE
Some changes in project config file to prevent "no repository found" errors on installation

### ISSUE THREAD

**Issue by mmueller013** [LorenzoBettini/packtpub-xtext-book-2nd-examples#38]
I commented out the following lines to prevent "no repository found" errors during installation with eclipse 2020-03:

line 146-147
```
      <!--repository
          url="https://download.eclipse.org/egit/updates"/-->

```
line 204- 205
```
      <!--repository
          url="http://download.eclipse.org/egit/github/updates"/-->
```

**Comment by LorenzoBettini** [comment]
This one `https://download.eclipse.org/egit/updates` works for me, but probably it's not needed anymore since its plugins are taken from the main Eclipse update site.
@mmueller013 would you mind providing a PR? Better to remove them, than to comment them.
Thanks in advance

**Comment by mmueller013** [comment]
Hi Lorenzo,
I cloned your Repository, and tried to reproduce the errors, to provide detailed information to the commit. Unfortunately, I cannot reproduce any "no repository found" errors today. I removed eclipse and the eclipse installer completely from my computer, and deleted any eclipse related directories from %TEMP% and %USER%. But no errors today ...
I have absolutely no explanation for that. I'm sorry for that.
Best regards Matthias
